Workbooks/Azure Update Management/Azure Update Management.workbook

```diff
@@ -0,0 +1,668 @@
+        "groupType": "editable",
+        "title": "Machines configurations reporting to selected workspaces",
+        "expandable": true,
+        "items": [
+          {
+            "type": 9,
+            "content": {
+              "version": "KqlParameterItem/1.0",
+              "parameters": [
+                {
+                  "id": "b2b56e4b-c365-4473-8346-84622e7485c8",
+                  "version": "KqlParameterItem/1.0",
+                  "name": "name",
+                  "label": "Computer name",
+                  "type": 1,
+                  "description": "Type computer name to filter",
+                  "value": "",
+                  "timeContext": {
+                    "durationMs": 86400000
+                  }
+                }
+              ],
+              "style": "formHorizontal",
+              "queryType": 0,
+              "resourceType": "microsoft.operationalinsights/workspaces"
+            },
+            "customWidth": "100",
+            "name": "parameters - 3",
+            "styleSettings": {
+              "maxWidth": "100%"
+            }
+          },
+          {
+            "type": 3,
+            "content": {
+              "version": "KqlItem/1.0",
+              "query": "Heartbeat \r\n| where Computer contains '{name}'\r\n| summarize arg_max(TimeGenerated, SourceComputerId, _ResourceId, Solutions) by Computer\r\n| summarize \r\nup = countif(Solutions has 'updates'),\r\nnup = countif(Solutions !has 'updates')\r\n| project machinePack = pack(\"Enabled\", up, \"Not enabled\", nup)\r\n| mv-expand machinePack\r\n| extend machine = tostring(bag_keys(machinePack)[0])\r\n| extend count_ = tolong(machinePack[machine])\r\n| project machine, count_ ",
+              "size": 0,
+              "title": "Machines reporting to chosen workspaces on which update solution is enabled vs not enabled",
+              "timeContext": {
+                "durationMs": 86400000
+              },
+              "queryType": 0,
+              "resourceType": "microsoft.operationalinsights/workspaces",
+              "crossComponentResources": [
+                "{Workspaces}"
+              ],
+              "visualization": "piechart",
+              "chartSettings": {
+                "showLegend": true
+              }
+            },
+            "customWidth": "50",
+            "name": "query - 3",
+            "styleSettings": {
+              "maxWidth": "50%"
+            }
+          },
+          {
+            "type": 3,
+            "content": {
+              "version": "KqlItem/1.0",
+              "query": "Heartbeat \r\n| extend IsEnabled = iff(Solutions has 'updates', 'Yes', 'No') \r\n| where Computer contains '{name}'\r\n| summarize arg_max(TimeGenerated, OSType, IsEnabled) by _ResourceId",
+              "size": 0,
+              "title": "Machines reporting to Log analytics with Updates solution enabled vs not enabled.",
+              "timeContext": {
+                "durationMs": 86400000
```

Comment:
Will you need a timerange parameter for this workbook or will it be defaulted a time for all custoemrs?